.gitignore

```diff
@@ -10,3 +10,15 @@ local.cfg
 .idea/
 node_modules
 .DS_Store
+/service/static/css/bootstrap-jisc.min.css
+/service/static/css/muk.APCRenderer.min.css
+/service/static/css/muk.funder.min.css
+/service/static/css/muk.institution.min.css
+/service/static/css/muk.publisher.min.css
+/service/static/css/muk.search.min.css
+/service/static/css/service.min.css
+/service/static/js/muk.funder.min.js
+/service/static/js/muk.institution.min.js
+/service/static/js/muk.min.js
+/service/static/js/muk.publisher.min.js
+/service/static/js/muk.search.min.js
```

Comment:
You should be able to just comment them out with a leading '#' so they can easily be reinstated later.